+adi/+FMComms5/Tx.m

```diff
@@ -0,0 +1,229 @@
+classdef Tx < adi.FMComms5.Base & adi.AD9361.Tx
+    % adi.FMComms5.Tx Transmit data from the FMComms5
+    %   The adi.FMComms5.Tx System object is a signal sink that can tranmsit
+    %   complex data from the FMComms5.
+    %
+    %   tx = adi.FMComms5.Tx;
+    %   tx = adi.FMComms5.Tx('uri','192.168.2.1');
+    %
+    %   <a href="https://www.analog.com/en/design-center/evaluation-hardware-and-software/evaluation-boards-kits/eval-ad-fmcomms5-ebz.html">Product Page</a>
+    
+    properties
+        %CenterFrequency Center Frequency
```

Comment:
Update all 3 parameters to include "Chip B" in docstrings. So CenterFrequency Center Frequency -> CenterFrequencyChipB Center Frequency Chip B